kratos/geometries/triangle_3d_3.h

```diff
@@ -867,7 +869,34 @@ template<class TPointType> class Triangle3D3
         const double Tolerance = std::numeric_limits<double>::epsilon()
         ) override
     {
-        PointLocalCoordinates( rResult, rPoint );
+        // The center of the geometry
+        const auto& r_center = this->Center();
+
+        // Tangent vectors
+        const array_1d<double, 3> tangent_xi  = this->GetPoint(1) - this->GetPoint(0);
+        const array_1d<double, 3> tangent_eta = this->GetPoint(2) - this->GetPoint(0);
+
+        // We compute the normal to check the normal distances between the point and the triangles, so we can discard that is on the triangle
+        array_1d<double, 3> normal;

```

Comment:
This is copy-paste of the `Normal` implemented in this class, please remove the duplication by using the function